packages/demo/src/common/Collection/computed/visibleKeys.js

```diff
@@ -0,0 +1,26 @@
+import {Computed} from 'cerebral'
+import paths from '../paths'
+import sort from '../sort'
+
+export default function (moduleName) {
+  const {collectionPath, draftPath, filterPath} = paths(moduleName)
+
+  return Computed(
+    {
+      items: `${collectionPath}.**`,
+      afilter: filterPath,
+      selectedKey: `${draftPath}.key`
+    },
+    ({items, afilter, selectedKey}) => {
+      const filter = afilter && afilter.toLowerCase()
+      const list = Object.keys(items).filter(key => (
+        !filter || items[key].name.toLowerCase().indexOf(filter) >= 0
+      ))
+      if (selectedKey && list.indexOf(selectedKey) < 0) {
```

Comment:
Hi Gaspard, when you have draft project on your page and you click the project selector (next to Start button), visibleProjectsByClient computed throws an error like `Uncaught TypeError: Cannot read property 'clientKey' of undefined(…)`. :(
I haven't check the code completely yet but when I remove this line, it does not show draft project. I understood that project list and project selector use visibleKeys computed. Also they are using the same filter. Maybe we should create a new computed for project selector. What do you think?